apps/docs/src/app/fn/component-docs/utilities/fn-disabled/fn-disabled-docs.component.html

```diff
@@ -0,0 +1,52 @@
+<fd-docs-section-title id="simple" componentName="FnDisabled">Basic example</fd-docs-section-title>
+<description>
+    <p>
+        <code>[fnDisabled]</code> is implementation of default view modifier. When passed true or used as an attribute
+        to <b>any</b> element, which is rendered in DOM will have similar traits. <code>[fnDisabled]</code> will add
+        <code>.is-disabled</code>, <code>[disabled]</code> and <code>[aria-disabled]</code> to the target element.
+    </p>
+    <p>
+        <code>[fnDisabled]</code> is independent directive and parent or child directives do not have any impact on it.
+        Everything is quite opposite.
```

Comment:
```suggestion
        <code>[fnDisabled]</code> is an independent directive and the parent or child directives do not have any impact on it.
```

Reply:
but fnDisabled has effect on other components in DI tree. Should we skip that scenario here?